### PR TITLE
Feature/issue#4 create writer skeleton

### LINKED ISSUE #4: Create writer skeleton
Implement writer proxy, wrap with docker image

### PR DESCRIPTION
**Pull request by YeaaaaH**
## Github

* [Github issue](https://github.com/ITA-Dnipro/Java-Home-Project-CQRS-Javelin/issues/4)

## Code reviewers

- [ ] @gendalf-fx 
- [ ] @vladyslavyarets 
- [ ] @Besuch 
- [ ] @paulspam 
- [ ] @Pavlo-Ponomarenko 

## Summary of issue

Implement writer from scratch

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: writer/src/main/java/com/softserveinc/ita/homeproject/writer/api/CommonApi.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
+package com.softserveinc.ita.homeproject.writer.api;
```
Comment (gendalf-fx) [inline writer/src/main/java/com/softserveinc/ita/homeproject/writer/api/CommonApi.java]: Delete unnecessary class, it seems like this class is never used.

File: writer/src/main/resources/application.properties [hunk 2]
```diff
@@ -0,0 +1,4 @@
+spring.datasource.url=${DATASOURCE_URL:jdbc:postgresql://host.docker.internal:5433/postgres}
+spring.datasource.username=${DATASOURCE_USER:postgres}
+spring.datasource.password=${DATASOURCE_PASSWORD:password}
+spring.datasource.driver-class-name=org.postgresql.Driver
```
Comment (gendalf-fx) [inline writer/src/main/resources/application.properties]: Add new line at the end of the file.

File: writer/src/test/java/com/softserveinc/ita/homeproject/writer/WriterApplicationTests.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class WriterApplicationTests {
+
+    @Test
+    void contextLoads() {
+    }
```
Comment (gendalf-fx) [inline writer/src/test/java/com/softserveinc/ita/homeproject/writer/WriterApplicationTests.java]: It seems like there is no tests.

File: writer-data-migration/pom.xml [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+                        <phase>package</phase>
+                        <goals>
+                            <goal>dockerBuild</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+</project>
```
Comment (gendalf-fx) [inline writer-data-migration/pom.xml]: Add new line at the end of the file.

File: writer/pom.xml [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+                        <phase>package</phase>
+                        <goals>
+                            <goal>dockerBuild</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+</project>
```
Comment (gendalf-fx) [inline writer/pom.xml]: Add new line at the end of the file.

**Review by gendalf-fx — CHANGES_REQUESTED** [review]

**Review by gendalf-fx — APPROVED** [review]

**Review by Besuch — APPROVED** [review]

**Pull request merged**